src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/BAU/UserManagement/UserInviteController.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationErrorMessages;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.BAU
+{
+    [Route("api")]
+    [ApiController]
+    [Authorize(Policy = "CanManageUsersOnSystem")]
+    public class UserInviteController : ControllerBase
```

Comment:
I've created https://dfedigital.atlassian.net/browse/EES-727 to test this as the current test is commented and I will remove it in the mean time.